datasquare_test.go

```diff
@@ -420,6 +422,47 @@ func BenchmarkEDSRoots(b *testing.B) {
 	}
 }
 
+func BenchmarkEDSRootsWithErasuredNMT(b *testing.B) {
+	ODSSizeByteUpperBound := 1024 * 1024 * 1024 // converting 512 MB to bytes
+	totalNumberOfShares := float64(ODSSizeByteUpperBound) / shareSize
+	// the closest power of 2 of the square root of
+	// the total number of shares
+	oDSShareSizeUpperBound := math.Pow(2, math.Ceil(math.Log2(math.Sqrt(
```

Comment:
Yes, `oDSShareSizeUpperBound` refers to the size of ODS in terms of number of shares in each row/column.

Reply:
okay then I think the variable should be renamed because we use `shareSize` elsewhere to describe the size of a share in bytes. We use `squareSize` to describe the number of shares in each row/column of the ODS.

Reply:
Sure, I revised it in https://github.com/celestiaorg/rsmt2d/pull/304/commits/d9b113064ba838abe0708ae56be930658c732b17 to resolve the confusion